Nktchinh

ntkchinh opened the pull request.

ntkchinh committed:
Create dot_product_formula.js

ntkchinh committed:
Update lib.js

ntkchinh committed:
change size of svg.dot_product_formula*

ntkchinh committed:
Create dot_product_formula2d.js

ntkchinh committed:
Update dot_product_formula.js

scaling and origin_position and color at table;

ntkchinh committed:
Update lib.js

remove arg start_coord_z

thtrieu commented on assets/js/linear_algebra/dot_product_formula.js:
```diff
@@ -0,0 +1,256 @@
+  let uTvv_line = [
+      {x: 0, y: 0, z: 0},
+      {x: uTvv.x, y: uTvv.y, z: uTvv.z,
+       color: 0, tt: true}
```
set uTvv_line.color = 0 instead

ntkchinh replied:
done

thtrieu commented on assets/js/linear_algebra/dot_product_formula.js:
```diff
@@ -0,0 +1,256 @@
+}
+
+function init(){
+  scatter = [];
```
there is no need for this line.

ntkchinh replied:
done

thtrieu commented on assets/js/linear_algebra/dot_product_formula.js:
```diff
@@ -0,0 +1,256 @@
+};
+
+})();
+
+
+
```
rm

ntkchinh replied:
done

thtrieu commented on assets/js/linear_algebra/dot_product_formula2d.js:
```diff
@@ -0,0 +1,273 @@
+  startAngleX = Math.PI,
+  startAngleY = 0.,
+  startAngleZ = 0.;
+let  axis_len = 2;
+
+let unit = axis_len/10;
```
merge these two lines into the same "let"?

ntkchinh replied:
done

thtrieu commented on assets/js/linear_algebra/dot_product_formula2d.js:
```diff
@@ -0,0 +1,273 @@
+  );
+
+  lib.sort();
+
```
rm

ntkchinh replied:
done

thtrieu commented on assets/js/linear_algebra/dot_product_formula2d.js:
```diff
@@ -0,0 +1,273 @@
+  drag_end();
+
+  text = [];
+
```
rm

ntkchinh replied:
done

thtrieu commented on assets/js/linear_algebra/dot_product_formula2d.js:
```diff
@@ -0,0 +1,273 @@
+       1000);
+  drag_end();
+
+  text = [];
```
rm

thtrieu replied:
what is this "text" variable?

ntkchinh replied:
done

thtrieu commented on assets/js/linear_algebra/lib.js:
```diff
@@ -618,45 +618,42 @@ function create_segments(d, k=10) {
-        row_coords.push(row_coords[i-1] +
-                           row_unit * rows_width_array[i-1]);
-    }
+                            cols_width_array, rows_width_array
```
better names: `col_heights` and `row_widths`

thtrieu replied:
or shorter names: `dhs` and `dws` (also sound more mathy)

ntkchinh replied:
done

thtrieu commented on assets/js/linear_algebra/dot_product_formula2d.js:
```diff
@@ -0,0 +1,273 @@
+                  drag_end_fn=drag_end);
+
+  // setting up the texts need to show;
+
```
rm

thtrieu commented on assets/js/linear_algebra/lib.js:
```diff
@@ -618,45 +618,42 @@ function create_segments(d, k=10) {
-        list_of_text.push(text_to_plot);
-      };
-    }
+  var list_of_text = [];
```
`list_of_texts`

ntkchinh committed:
Update lib.js

changing var name

ntkchinh committed:
Update lib.js

change all 'var' to 'let'

ntkchinh committed:
Update dot_product_formula.js

change var to let;
drop empty lines;
using uTvv_line.color;
change var name in function text_table_to_list;

ntkchinh committed:
Update dot_product_formula2d.js

'var' to 'let';
drop empty lines;
using uTvv_line.color;
change var name in text_table_to_list;

ntkchinh committed:
Update dot_product_project.js

'var' to 'let';
replace scatter[*] with u & v;
showing uTvv;

ntkchinh committed:
Update dot_product_project2d.js

'var' to 'let';
replace scatter[*] with u, v;
changing var name (projection_line, dot_product..);
plot uTvv;

thtrieu committed:
Merge branch 'master' into nktchinh
